core/src/main/java/io/cloudevents/core/CloudEventUtils.java

```diff
@@ -102,4 +102,16 @@ public static CloudEvent toEvent(CloudEventReader reader, @Nullable CloudEventDa
         return mapper.map(data);
     }
 
+    /**
+     * Validates the extension name as defined in  CloudEvents spec
```

Comment:
Can you add a link to the spec here: https://github.com/cloudevents/spec/blob/master/spec.md#attribute-naming-convention